src/Chapter1.hs

```diff
@@ -632,7 +644,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = 
+    let mod10 = mod (abs n) 10
+        mod100 = mod (abs n) 100
+    in div (mod100 - mod10) 10 + mod10
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂